openff/benchmark/analysis/new_draw.py

```diff
@@ -0,0 +1,973 @@
+            )
+        else:
+            g.map(
+                sns.kdeplot,
+                x_label,
+                clip_on=False,
+                shade=True,
+                alpha=0.5,
+                lw=ridgedict["lw"],
+                bw=bw,
+            )
+
+        # draw colored horizontal line below densities
+        g.map(plt.axhline, y=0, lw=ridgedict["lw"], clip_on=False)
+
+    else:
+
+        # draw black horizontal line below densities
+        plt.axhline(y=0, color="black")
+        
+        # draw outline around densities; can also single outline color: color="k"
+    if bw == "hist":
+        histoptions = {
+            "element": "step",
+            "alpha": 1.0,
+            "linewidth": ridgedict["lw"],
+            "fill": False,
+            "binrange": hist_range,
+        }
+        g.map(
+            sns.histplot,
+            x_label,
+            bins=15,
+            stat="count",
+            **histoptions,
+        )
+
+    else:
+        g.map(sns.kdeplot, x_label, clip_on=False, lw=ridgedict["lw"], bw=bw)
+
+    # draw a vertical line at x=0 for visual reference
+    g.map(plt.axvline, x=0, lw=ridgedict["vl"], ls="--", color="gray", clip_on=False)
+
+    # optional: add custom vertical line
+    # g.map(plt.axvline, x=0.12, lw=1, ls='--', color='gray', clip_on=False)
+
+    # add labels to each level
+    if not same_subplot:
+        g.map(label, x_label)
+
+    # else if single subplot, generate a custom legend
+    else:
+        cmap = mpl.cm.tab10
+        patches = []
+        for i, label in enumerate(results):
+            patches.append(
+                mpl.patches.Patch(
+                    color=cmap(i/10),
+                    label=label,
+                )
+            )
+        plt.legend(handles=patches, fontsize=ridgedict["xfontsize"] / 1.2)
+    
+        # optional: set symmetric log scale on x-axis
+    if sym_log:
+        g.set(xscale="symlog")
+
+    # Set the subplots to overlap
+    if not same_subplot:
+        g.fig.subplots_adjust(hspace=-0.45)
+    else:
+        g.fig.subplots_adjust(hspace=-1.0)
+    
+    # Remove axes details that don't play well with overlap
+    g.set_titles("")
+    #    g.set(yticks=[])
+    # g.despine(bottom=True, left=True)
+    # ax = plt.gca()
+    # ax.spines['left'].set_visible(True)
+    # ax.spines['left'].set_position('zero')
+    # ax.set_yticks([0.4])
+    if what_for == "paper":
+        plt.gcf().set_size_inches(7, 3)
+    elif what_for == "talk":
+        plt.gcf().set_size_inches(12, 9)
+
+    # adjust font sizes
+    plt.xlabel(x_label, fontsize=ridgedict["xfontsize"])
+    plt.ylabel("Count", fontsize=ridgedict["xfontsize"])
+    plt.xticks(fontsize=ridgedict["xfontsize"])
+    #plt.yticks(fontsize=ridgedict["xfontsize"])
+
+    # save with transparency for overlapping plots
+    plt.savefig(out_file, transparent=True, bbox_inches="tight")
+    plt.clf()
+
+
+def plot_lucas(results_dir, output_directory):
+    global results
```

Comment:
Remove use of `global`. Shouldn't be necessary and we really don't want global state on the module.